### ISSUE TITLE
Fine-tune on custom fisheye dataset

### ISSUE THREAD

**Issue by aparico** [duanzhiihao/RAPiD#39]
@duanzhiihao 

I would like to fine-tune COCO-pretrained RAPiD on some custom fisheye dataset that is similar to CEPDOF, WEPDTOF, MW-R, and HABBOF.

Just to clarify, what I mean by fine-tune is to freeze the backbone and train only the neck and the head at a lower learning rate.

Could you please upload your fine-tuning functions / script? 🙏

Thank you.

**Comment by aparico** [comment]
I just wrote the code myself using this as a [guide](https://pytorch.org/docs/stable/notes/autograd.html?highlight=freeze#:~:text=To%20freeze%20parts%20of%20your%20model,which%20have%20requires_grad%3DTrue%20by%20default).)
